commit 87363770ea3fe6167445b6d445ca563f2a3f05fc
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1339430277 -0500

    Restore tdesu dialog "Keep password" check box default to disabled/unchecked.

diff --git a/tdesu/defaults.h b/tdesu/defaults.h
index 4649b2b..7fd724e 100644
--- a/kdesu/defaults.h
+++ b/kdesu/defaults.h
@@ -13,9 +13,8 @@
 #ifndef __Defaults_h_included__
 #define __Defaults_h_included__
 
-/*const int defTimeout = 120*60;*/
 const int defTimeout = 120*60;
 const int defEchoMode = 0;
-const int defKeep = true;
+const int defKeep = false;
 
 #endif
